### DISCUSSION TITLE
Open Source Reconciliation

### DISCUSSION THREAD

**Discussion by euluizaraujo** [juspay/hyperswitch#3063] [Ideas / Feature Requests]
Hello! Is there any plan or possibility of making the reconciliation functionality open source in December? We are getting ready to use the self-hosted version and this functionality would be very useful.

**Comment by SanchithHegde** [comment]
Hey @euluizaraujo, I'm afraid open sourcing the reconciliation functionality is not on our roadmap for this quarter. You would have to develop the functionality yourself or use the hosted version if it is immediately required.

**Reply by swl0732111** [reply]
Could you please share some idea by using what technical stack to achieve this, e.g. Flink or etc. Thx

**Comment by muditbhutani** [comment]
@euluizaraujo were you able to find a relevant open source reconciliation software for your use case

**Comment by muditbhutani** [comment]
Hey @euluizaraujo we have a reconciliation tool that we plan to open source but do not have a defined timeline yet

Documentation here: https://docs.hyperswitch.io/about-hyperswitch/payments-modules/reconciliation-product

If you would like to be part of the beta, we can help setup the repo and reconciliation for you in your system, we are only taking a couple of merchants for this to figure out deployment issues before we open to everyone
